starcheck/src/starcheck.pl

```diff
@@ -1223,3 +1223,5 @@ =head1 AUTHOR
 
 =cut
 
+exit 0
```

Comment:
I think this should be at the end of the END {} block above the docs.

Reply:
The problem with that, I think, is that END still runs after an unexpected "die", so you get zero exit status even on a failure.

Reply:
Ah good point. Sorry, never mind.

Reply:
I think maybe
```
END {
   my $exit_status = $?;
   ...
   exit($exit_status);
}
```